src/Caching/SqlServer/src/SqlServerCacheOptions.cs

```diff
@@ -43,6 +43,11 @@ public class SqlServerCacheOptions : IOptions<SqlServerCacheOptions>
         /// </summary>
         public TimeSpan DefaultSlidingExpiration { get; set; } = TimeSpan.FromMinutes(20);
 
+        /// <summary>
+        /// A function, if provided, should return an access token for authentication
```

Comment:
```suggestion
        /// Gets or sets an access token provider that will be called to return a token for each SQL connection.
```

Let's use similar language to https://github.com/aspnet/AspNetCore/blob/b1ca2c1155da3920f0df5108b9fedbe82efaa11c/src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnectionOptions.cs#L87